src/api/getBaldiyatsForDairaByCodeForWilayaByCode.js

```diff
@@ -0,0 +1,31 @@
+const projectBaladiya = require('../utils/projections/projectObject');
+
+const getBaldiyatsForDairaByCodeForWilayaByCode = (data) =>
```

Comment:
Same as my previous comments, plus this name can be simplified to `getBaldiyatsByWilayaCode` or `getBaldiyatsByDairaCode`. And if you could find a counterexample/argument to keep such composed methods which means that daira code is not enough to identify wilayas. we can name this: `getBaldiyatsByWilayaAndDairaCodes`